rcsb/workflow/wuw/ExDbWorkflow.py

```diff
@@ -0,0 +1,521 @@
+##
+# File: ExDbWorkflow.py
+# Date: 17-Dec-2019  jdw
+#
+#  Workflow wrapper  --  exchange database loading utilities --
+#
+#  Updates:
+#   2-Mar-2023 dwp Add "numProc" parameter to 'upd_ref_seq' operation methods
+#   9-Mar-2023 dwp Lower refChunkSize to 10 (UniProt API having trouble streaming XML responses)
+#  25-Apr-2024 dwp Add arguments and methods to support CLI usage from weekly-update workflow
+#  20-Aug-2024 dwp Add LoadTargetCofactors step; change name of UpdateTargetsCofactors step to UpdateTargetsData
```

Comment:
For this and other files `*.py`, can you please just add a comment noting that you moved these scripts from `rcsb.exdb`? (Appreciate what you've already noted in the HISTORY.txt!)